content/docs/azimuth/l2/layer2.md

```diff
@@ -0,0 +1,402 @@
+and your sponsor/sponsee are not mixing layer 1 and layer 2 sponsorship actions
+between yourselves, then you have nothing to worry about and may safely ignore this
+section.
+
+But, for instance, if both `~sampel-palnet` and `~dopzod` are on layer 1, it is
+technically possible for `~sampel-palnet` to escape to `~dopzod` on layer 1, and
+then `~dopzod` can accept the escape on layer 2. This will result in `~dopzod`
+appearing as the sponsor in the Layer 2 Azimuth state (and thus be
+`~sampel-palnet`'s "true" sponsor), and the sponsor in the Layer 1 Azimuth state
+will remain unchanged. While it is difficult to imagine a good reason to do
+this, developers working with layer 2 need to keep in mind these edge cases and
+ought to read on.
+
+In the following table, columns `E_1` and `S_1` represent the escape status and
+sponsor of a given ship as reflected by the Layer 1 Azimuth state. Columns `E_2`
+and `S_2` represent the escape status and sponsor of a given ship as reflected
+in the Layer 2 Azimuth State. The "true" escape status and sponsor of a ship is
+always what is listed in the Layer 2 Azimuth state. In other words, at the end
+of the day, `S_2` is always the sponsor that matters, but layer 1 actions can
+affect the values of `E_2` and `S_2`.
+
+A tar `*` entry represents any value and if an event shows a transition
+from `*` to `*` that means that value is not altered by the transition. The
+transitions marked with `!!` are prohibited by the layer 1 Azimuth smart
+contract and thus never occur. `A1` and `A2` represent two distinct ships.
+```
+Event        | E_1 | E_2 | S_1 | S_2 | -> | E_1 | E_2 | S_1 | S_2
+L1-escape A1 | *   | *   | *   | *   | -> | A1  | A1  | *   | *
+L1-cancel A1 | ~   | *   | *   | *   | -> !! :: no cancel if not escaping
+L1-cancel A1 | A1  | *   | *   | *   | -> | ~   | ~   | *   | *
+L1-adopt  A1 | A1  | *   | *   | *   | -> | ~   | ~   | A1  | A2
+L1-adopt  A1 | ~   | *   | *   | *   | -> !! :: no adopt if not escaping
+L1-adopt  A1 | A2  | *   | *   | *   | -> !! :: no adopt if not escaping
+L1-detach A1 | *   | *   | A1  | A1  | -> | *   | *   | ~   | ~
+L1-detach A1 | *   | *   | A1  | A2  | -> | *   | *   | ~   | A2
+L1-detach A1 | *   | *   | A1  | ~   | -> | *   | *   | ~   | ~
+L2-escape A1 | *   | *   | *   | *   | -> | *   | A1  | *   | *
+L2-cancel A1 | *   | *   | *   | *   | -> | *   | ~   | *   | *
+L2-adopt  A1 | *   | A1  | *   | *   | -> | *   | ~   | *   | A1
+L2-adopt  A1 | *   | A2  | *   | *   | -> | *   | A2  | *   | *
+L2-adopt  A1 | *   | ~   | *   | *   | -> | *   | ~   | *   | *
+L2-reject A1 | *   | A1  | *   | *   | -> | *   | ~   | *   | *
+L2-reject A1 | *   | A2  | *   | *   | -> | *   | A2  | *   | *
+L2-reject A1 | *   | ~   | *   | *   | -> | *   | ~   | *   | *
+L2-detach A1 | *   | *   | *   | A1  | -> | *   | *   | *   | ~
+L2-detach A1 | *   | *   | *   | A2  | -> | *   | *   | *   | A2
+L2-detach A1 | *   | *   | *   | ~   | -> | *   | *   | *   | ~
+```
```

Comment:
Tried it as a markdown table. I think it looks a bit worse but neither is perfect https://urbit-org-n8olznehl-urbit.vercel.app/docs/azimuth/l2/layer2

It ends up taller so in my current browser window I can't see the entire thing at once. But if I move my browser window to a vertical slot (I'm using a tiling WM) then the code block one is too wide and the markdown one is fine. But I think the lack of dividers just makes it a bit more ugly, and the monospace font is cleaner. Either one leaves something to be desired, but I have a slight preference for the code block since its easier to copy and paste into an editor.